### PR TITLE
standardify

### PR DESCRIPTION
**Pull request by dickeyxxx**
I don't really expect anyone to actually *review* this, mostly just assume it's all good :)

### CODE REVIEW

File: commands/addons/attach.js [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
 
-function* run (context, heroku) {
-  let app = context.app;
-  let addon = yield heroku.get(`/addons/${context.args.addon_name}`);
+function * run (context, heroku) {
+  let app = context.app
+  let addon = yield heroku.get(`/addons/${context.args.addon_name}`)
 
-  function createAttachment(app, as, confirm) {
+  function createAttachment (app, as, confirm) {
```
Comment (bjeanes) [inline commands/addons/attach.js]: wow... really? That seems surprisingly unconventional in basically any C-style lang

File: commands/addons/attach.js [hunk 2]
```diff
@@ -1,44 +1,44 @@
...
     return cli.action(
       `Attaching ${cli.color.addon(addon.name)}${as ? ' as ' + cli.color.attachment(as) : ''} to ${cli.color.app(app)}`,
       heroku.request({
         path: '/addon-attachments',
         method: 'POST',
         body: {
-          name:  as,
-          app:   {name: app},
+          name: as,
+          app: {name: app},
```
Comment (bjeanes) [inline commands/addons/attach.js]: booooo

File: commands/addons/attach.js [hunk 3]
```diff
@@ -48,9 +48,9 @@ module.exports = {
   needsAuth: true,
   needsApp: true,
   flags: [
-    {name: 'as',      description: 'name for add-on attachment', hasValue: true},
-    {name: 'confirm', description: 'overwrite existing add-on attachment with same name', hasValue: true},
```
Comment (bjeanes) [inline commands/addons/attach.js]: ugh trailing `,` makes copy/paste and re-ordering so much nicer though

**Comment by bjeanes** [comment]
@dickeyxxx I think standard (the tool) is making some choices that I personally wouldn't, but :+1: on having *some* standard :)

File: commands/addons/attach.js (hunk 3 above)
Comment (dickeyxxx) [inline commands/addons/attach.js]: Yes but so does not having semicolons. Win some lose some!

File: commands/addons/attach.js (hunk 1 above)
Comment (dickeyxxx) [inline commands/addons/attach.js]: that wasn't @hunterloftis's favorite setting either

**Comment by dickeyxxx** [comment]
yep, I think that's the way it's supposed to be though, there's no way an off-the-shelf unconfigurable tool is going to be right for everyone, but definitely good to have a linter this opinionated. A lot of stuff like no trailing commas isn't how I would write the code, but I get why. It's because it makes it so there is only one way to write something. And having trailing commas all the time would be terrible!

```js
let arr = [1,];
```

**Pull request merged**